### ISSUE TITLE
is it possible to present result as cloud ?

### ISSUE THREAD

**Issue by om35** [MaartenGr/BERTopic#425]
is it possible with Topic Bert  to present result (topics and words) as cloud like this ? :
<img width="764" alt="cloud" src="https://user-images.githubusercontent.com/45729715/151770685-4f58a6de-e774-4d0b-aa48-414cd424bbc0.png">

size of word = frequency 
if yes ? how we can do this ? with which tools ?  at what level ? any ideas please

**Comment by MaartenGr** [comment]
You can typically achieve this with the [wordcloud](https://github.com/amueller/word_cloud) package. You can find a minimal example of working with frequencies [here](https://amueller.github.io/word_cloud/auto_examples/frequency.html#sphx-glr-auto-examples-frequency-py). 

So something like this:

```python
from sklearn.datasets import fetch_20newsgroups
from bertopic import BERTopic
from wordcloud import WordCloud
import matplotlib.pyplot as plt

def create_wordcloud(topic_model, topic):
    text = {word: value for word, value in topic_model.get_topic(topic)}
    wc = WordCloud(background_color="white", max_words=1000)
    wc.generate_from_frequencies(text)
    plt.imshow(wc, interpolation="bilinear")
    plt.axis("off")
    plt.show()

# Train topic model
docs = fetch_20newsgroups(subset='all',  remove=('headers', 'footers', 'quotes'))['data']
topic_model = BERTopic(verbose=True)
topics, probs = topic_model.fit_transform(docs)

# Show word cloud
create_wordcloud(topic_model, topic=1)

**Comment by MaartenGr** [comment]
Closing this for now. However, if you want to discuss this further, let me know and I'll make sure to reopen the issue!

**Comment by AbuBakarrr** [comment]
You are 😍ᒐ♡ᘎᙓ😍, Sir MaatenGr... such  a great support. 

💙💙💙💚💚💚💛💛💛💜💜💜

**Comment by rsg-iik** [comment]
Hi @MaartenGr,

Thanks a lot for creating this awesome package!
I am able to get really valuable insights for my use case using this.

I would like to create word cloud for all topics together so that this process can be automated. The above code #425(https://github.com/MaartenGr/BERTopic/issues/425) works for me and I am using this to get word clouds for all topics (e.g. 10 topics generate by BERTopic)as:

for topic in range(0,10):
      create_wordcloud(topic_model, topic)

Is there any way we could extract all topics together from topic_model instead of manually passing number of topics to create_wordcloud functions? (Topc2vec has this ability to generate word clouds for all topic together). Please let me know.  
Thanks

**Comment by MaartenGr** [comment]
@rsg-iik That is currently not implemented as it would require adding another dependency to BERTopic. For that reason, I added how to create a word cloud to the documentation for the new release with instructions on how to install the additional dependency.

**Comment by rsg-iik** [comment]
@MaartenGr Thanks for letting me know!
 I really appreciate your prompt response. I look forward to using new version.
